### PR TITLE
Don't close a connection that was not open.

### PR DESCRIPTION
**Pull request by dennisgranath**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -86,9 +85,11 @@ func update(config *Config) {
 				"StatusCode": statuscode,
 			}).Error("Unable to create server entry")
 		} else {
+			resp.Body.Close()
 			cxtLog.Info("Server entry created")
 		}
 	} else {
+		resp.Body.Close()
```
Comment (bromid) [inline main.go]: Might leak connections:
http://devs.cloudimmunity.com/gotchas-and-common-mistakes-in-go-golang/index.html#close_http_resp_body

**Comment by dennisgranath** [comment]
shipit

**Comment by bromid** [comment]
shipit

**Comment by dennisgranath** [comment]
lgtm

**Pull request merged**